### ISSUE TITLE
Hyperparameters for getting pre-trained shufflenet

### ISSUE THREAD

**Issue by d-li14** [tensorpack/tensorpack#1178]
I noticed that your pre-trained ShuffleNet V1 / V2 got high accuracy close to the original ones in the paper. Could you please share the specific training settings to reproduce? Thanks.

**Comment by ppwwyyxx** [comment]
The command to reproduce is written in the readme.

**Comment by ppwwyyxx** [comment]
> To train any of the models, just do ./{model}.py --data /path/to/ilsvrc. More options are available in ./{model}.py --help.

"More options" for shufflenet is in the results table.

**Comment by d-li14** [comment]
Thanks, got it! I will try.
